yt_dlp/extractor/stacommu.py

```diff
@@ -125,24 +149,83 @@ class StacommuLiveIE(StacommuBaseIE):
         'params': {
             'skip_download': 'm3u8',
         },
+    }, {
+        'url': 'https://www.stacommu.jp/en/live/d2FJ3zLnndegZJCAEzGM3m',
+        'only_matching': True,
     }]
 
     _API_PATH = 'events'
 
     def _real_extract(self, url):
-        video_id = self._match_id(url)
-        video_info = self._call_api(video_id, msg='video information', query={'al': 'ja'}, auth=False)
-        hls_info, decrypt = self._call_encrypted_api(
-            video_id, ':watchArchive', 'stream information', data={'method': 1})
+        return self._extract_ppv(url)
 
-        return {
-            'id': video_id,
-            'formats': self._get_formats(hls_info, ('hls', 'urls', ..., {url_or_none}), video_id),
-            'hls_aes': self._extract_hls_key(hls_info, 'hls', decrypt),
-            **traverse_obj(video_info, {
-                'title': ('displayName', {str}),
-                'timestamp': ('startTime', {int_or_none}),
-                'thumbnail': ('keyVisualUrl', {url_or_none}),
-                'duration': ('duration', {int_or_none}),
-            }),
-        }
+
+class TheaterComplexTownBaseIE(StacommuBaseIE):
+    _NETRC_MACHINE = 'theatercomplextown'
+    _API_HOST = 'api.theater-complex.town'
+    _LOGIN_QUERY = {'key': 'AIzaSyAgNCqToaIz4a062EeIrkhI_xetVfAOrfc'}
+    _LOGIN_HEADERS = {
+        'Accept': '*/*',
+        'Content-Type': 'application/json',
+        'X-Client-Version': 'Chrome/JsCore/9.9.4/FirebaseCore-web',
```

Comment:
While at it, should the client version be updated?
```suggestion
        'X-Client-Version': 'Chrome/JsCore/9.23.0/FirebaseCore-web',
```
Note: I didn't test this, but only looked at the JS.